cassdegrees/ui/templatetags/student_course_boxes.py

```diff
@@ -49,5 +49,5 @@ def index(List, i):
 
 
 @register.filter
-def listIndex(List, i):
-    return List[i]
+def get(List, i):
+    return List[str(i)]
```

Comment:
`str` or `int` here? If it is a string key maybe this should be specified as something like `get_by_string`?

Reply:
Yeah Django was being very strange here. Even when the `List` dictionary had a integer key, it would be converted to a string anyway and would break because it couldn't find the integer key. The reason this worked before was that `List` was an array rather than a dictionary, so ints were valid. I named it `get` to reference the fact that python's `get` function is only used in dictionaries. If you still think I need to be more specific then I will, but I think this is fine